_data/SoumyaChoubey/SoumyaChoubey.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Soumya Choubey
+institution: Government Engineering College ,Raipur,Chhattisgarh,India
+profile_pic: soumya.jpg
```

Comment:
Incorrect file name, please do one of the following:
1. Change `soumya.jpg` to `Soumya.jpg` in the markdown file.
2. Rename `Soumya.jpg` to `soumya.jpg`.
I'd suggest you rename the file!

Reply:
Cool doing in a min.

Reply:
> Incorrect file name, please do one of the following:
> 
> 1. Change `soumya.jpg` to `Soumya.jpg` in the markdown file.
> 2. Rename `Soumya.jpg` to `soumya.jpg`.
>    I'd suggest you rename the file!

Done!